webknossos/tests/cassettes/test_generated_client/test_user_list.yaml

```diff
@@ -17,49 +17,42 @@ interactions:
       minds","isTeamManager":true}],"experiences":{},"lastActivity":1583756665727,"isAnonymous":false,"isEditable":true,"organization":"scalable_minds","novelUserExperienceInfos":{"shouldSeeModernControlsModal":true,"hasSeenDashboardWelcomeBanner":true},"selectedTheme":"auto","created":1583756665727,"lastTaskTypeId":null},{"id":"5b4cdd6d1d0000864155f737","email":"haedngaga@gmail.com","firstName":"Haedn","lastName":"Rho","isAdmin":false,"isDatasetManager":false,"isActive":false,"teams":[{"id":"5b3ce78557b7414e59269427","name":"scalable
       minds","isTeamManager":false}],"experiences":{},"lastActivity":1531764077527,"isAnonymous":false,"isEditable":true,"organization":"scalable_minds","novelUserExperienceInfos":{"shouldSeeModernControlsModal":true,"hasSeenDashboardWelcomeBanner":true},"selectedTheme":"auto","created":1531764077529,"lastTaskTypeId":null},{"id":"5b5dd2fb1c00008230ec8174","email":"n@scm.io","firstName":"Norman","lastName":"Rzepka","isAdmin":true,"isDatasetManager":false,"isActive":true,"teams":[{"id":"5b3ce78557b7414e59269427","name":"scalable
-      minds","isTeamManager":false}],"experiences":{},"lastActivity":1532875515002,"isAnonymous":false,"isEditable":true,"organization":"scalable_minds","novelUserExperienceInfos":{"lastViewedWhatsNewTimestamp":1630414304516,"shouldSeeModernControlsModal":false,"hasSeenDashboardWelcomeBanner":true},"selectedTheme":"dark","created":1532875515006,"lastTaskTypeId":null},{"id":"5b8e93aa1e0000b90a4aa940","email":"robert.schwanhold@scalableminds.com","firstName":"Robert","lastName":"Schwanhold","isAdmin":true,"isDatasetManager":false,"isActive":true,"teams":[{"id":"5b3ce78557b7414e59269427","name":"scalable
+      minds","isTeamManager":false}],"experiences":{},"lastActivity":1532875515002,"isAnonymous":false,"isEditable":true,"organization":"scalable_minds","novelUserExperienceInfos":{"lastViewedWhatsNewTimestamp":1641906588957,"shouldSeeModernControlsModal":false,"hasSeenDashboardWelcomeBanner":true},"selectedTheme":"dark","created":1532875515006,"lastTaskTypeId":null},{"id":"5b8e93aa1e0000b90a4aa940","email":"robert.schwanhold@scalableminds.com","firstName":"Robert","lastName":"Schwanhold","isAdmin":false,"isDatasetManager":false,"isActive":false,"teams":[{"id":"5b3ce78557b7414e59269427","name":"scalable
       minds","isTeamManager":false}],"experiences":{},"lastActivity":1536070570797,"isAnonymous":false,"isEditable":true,"organization":"scalable_minds","novelUserExperienceInfos":{"shouldSeeModernControlsModal":true,"hasSeenDashboardWelcomeBanner":true},"selectedTheme":"auto","created":1536070570798,"lastTaskTypeId":null},{"id":"6006a44d0100007e0070d734","email":"jonathan@scm.io","firstName":"Jonathan","lastName":"Striebel","isAdmin":true,"isDatasetManager":false,"isActive":true,"teams":[{"id":"5b3ce78557b7414e59269427","name":"scalable
```

Comment:
I am surprised to still see firstName, lastName and email of real users in this public repository. In my mind, this is unacceptable (while without explicit user consent) and in direct violation of https://webknossos.org/privacy – please [take steps to remove those entries](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository).

Reply:
Good point, glad you noticed it! Not sure how this slipped through. For now I removed the entries in this list in this PR, we'll need to merge it first to master before fixing this with bfg since it does not change the master's HEAD.

Reply:
Thanks for looking into this! Maybe we could add an assertion that no fields of these names are left at some point, so that this does not happen again. Note that annotation jsons also contain user information, not sure if those are around here somewhere (may only be _your_ info, since they are your annotations, which maybe you are comfortable publishing)